demos/node-classification/deep-graph-info-max/gcn-deep-graph-info-max-cora-example.ipynb

```diff
@@ -0,0 +1,293 @@
+{
```

Comment:
Oh, another minor thing in addition to the spelling: this should be named `demos/node-classification/deep-graph-infomax/gcn-deep-graph-infomax-cora-example.ipynb`.

That is, replace both `info-max`s with `infomax`, matching the paper and the `InfoMax` -> `Infomax` renaming.